_component_design/search.md

```diff
@@ -113,46 +163,127 @@ The basic search bar component has the following states:
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Spacing
+### Elements
 
-- Content is vertically centered.
-- Padding around edges of search box: 8px.
-- When displaying an error message or "no search results" message, place 4px below the search bar.
+**Elements:** Assistive search boxes consist of an indicator bar, divider line, and assistance at minimum. Category headers and the actual kind of assistive text are to be determined by your product’s needs. You may also include “Additional attributes” to help the user identify the category of the search result.
 
 {% endcolumn %}
 
 {% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-
-{% figure [caption:"Spacing specifications"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/components/search-and-filters/search-bar/search-spacing.svg"/>
+{% figure [caption:"Assisted search container elements"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/search-and-filters/search-assisted/assisted-search-container-elements.png" width="788"/>
 {% endfigure %}
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<!-- Commented out because search pills were put on hold for this pattern. 2/6/18-EN
```

Comment:
What's the point in adding this if it's just commented out? Can this be placed in its own file, to be brought in at a future point?